restx-factory-annotation-processor compatibility with Java 11

*Jdk version*: java-1.8.0-openjdk-1.8.0.242.b08-0.fc31.x86_64 & java-11-openjdk-11.0.6.10-0.fc31.x86_64
*Maven version*: Maven plugin of Intellij Idea 2020.1 located in <idea_path>/plugins/maven/lib/maven3 -> Maven 3.6.1 (core)

**Stepts to reproduce:**
1. Checkout master branch (current commit: c92e983)
2. Perform a `mvn install` on `restx-parent` with jdk8 (so `restx-factory:0.36` is available as a dependency)
3. Perform a mvn verify on `restx-factory-annotation-processor` with jdk 11

**Expected behaviour:**
Dependencies are downloaded, application is packaged, tests are launched

**Current behaviour:**
Build fail for module `restx-factory-annotation-processor`
```
[INFO] 
[INFO] ------------< io.restx:restx-factory-annotation-processor >-------------
[INFO] Building restx-factory-annotation-processor 0.36-SNAPSHOT         [4/41]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-maven) @ restx-factory-annotation-processor ---
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ restx-factory-annotation-processor ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 6 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.7.0:compile (default-compile) @ restx-factory-annotation-processor ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 2 source files to /home/nhumblot/dev/wkspace/restx/restx-factory-annotation-processor/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/nhumblot/dev/wkspace/restx/restx-factory-annotation-processor/src/main/java/restx/factory/processor/FactoryAnnotationProcessor.java:[178,40] reference to Module is ambiguous
  both class restx.factory.Module in restx.factory and class java.lang.Module in java.lang match
[ERROR] /home/nhumblot/dev/wkspace/restx/restx-factory-annotation-processor/src/main/java/restx/factory/processor/FactoryAnnotationProcessor.java:[193,51] reference to Module is ambiguous
  both class restx.factory.Module in restx.factory and class java.lang.Module in java.lang match
[ERROR] /home/nhumblot/dev/wkspace/restx/restx-factory-annotation-processor/src/main/java/restx/factory/processor/FactoryAnnotationProcessor.java:[71,69] reference to Module is ambiguous
  both class restx.factory.Module in restx.factory and class java.lang.Module in java.lang match
[ERROR] /home/nhumblot/dev/wkspace/restx/restx-factory-annotation-processor/src/main/java/restx/factory/processor/FactoryAnnotationProcessor.java:[79,17] reference to Module is ambiguous
  both class restx.factory.Module in restx.factory and class java.lang.Module in java.lang match
[ERROR] /home/nhumblot/dev/wkspace/restx/restx-factory-annotation-processor/src/main/java/restx/factory/processor/FactoryAnnotationProcessor.java:[79,53] reference to Module is ambiguous
  both class restx.factory.Module in restx.factory and class java.lang.Module in java.lang match
[INFO] 5 errors 
[INFO] -------------------------------------------------------------
```

--- a/restx-factory-annotation-processor/src/main/java/restx/factory/processor/FactoryAnnotationProcessor.java
+++ b/restx-factory-annotation-processor/src/main/java/restx/factory/processor/FactoryAnnotationProcessor.java
@@ -6,7 +6,13 @@ import com.google.common.collect.*;
 import com.google.common.io.CharStreams;
 import com.samskivert.mustache.Template;
 import restx.common.processor.RestxAbstractProcessor;
-import restx.factory.*;
+import restx.factory.Alternative;
+import restx.factory.Component;
+import restx.factory.Machine;
+import restx.factory.Module;
+import restx.factory.NamedComponent;
+import restx.factory.Provides;
+import restx.factory.When;
 
 import javax.annotation.processing.RoundEnvironment;
 import javax.annotation.processing.SupportedAnnotationTypes;

